code/procs/accents.dm

```diff
@@ -834,12 +834,17 @@ proc/random_accent()
 	switch(S)
 
 		if("w") //germans pronounce W like V
-			new_string = "v"
-			used = 1
-		if("t") //unlinke the stereotypical zat- it's more like dat, because both the t and the h are pronounced in german
+			if(lowertext(R.next_char) == " " || lowertext(R.next_char) == null)
```

Comment:
Shouldn't this be the other way around? Currently it looks like it's *only* changing w's to v's on the end of words

Reply:
oop- probably. I'm very sleep-deprived. FIxing that now :P